pkg/metrics/azure_metrics_test.go

```diff
@@ -17,11 +17,51 @@ limitations under the License.
 package metrics
 
 import (
+	"bytes"
+	"flag"
 	"testing"
 
+	"github.com/go-logr/logr"
 	"github.com/stretchr/testify/assert"
+	"k8s.io/klog/v2"
```

Comment:
nit: please insert an empty line between imports from different orgs.

Reply:
Thanks for advice! I have modified the code and will pay attention to it in the future~